pkg/registries/set_impl.go

```diff
@@ -86,7 +89,7 @@ func (e *setImpl) GetRegistryMetadataByImage(image *storage.Image) *types.Config
 
 	reg := e.getRegistryByImageNoLock(image)
 	if reg != nil {
-		return reg.Config()
+		return reg.Config(context.Background())
```

Comment:
why not passing context to this method?

Reply:
same here, passing the parent contexts would involve a lot of functions.